guides/doc-Monitoring_Project/topics/proc_configuring-pcp-data-collection.adoc

```diff
@@ -0,0 +1,114 @@
+[id='configure-pcp-data-collection_{context}']
+= Configuring PCP Data Collection
+
+This procedure describes how to configure PCP to collect metrics about processes, {Project}, Apache HTTP Server, and PostgreSQL.
+
+.Procedure
+
+. To configure PCP to collect data about {Project} processes, create the `/var/lib/pcp/pmdas/proc/hotproc.conf` file and include the following content:
++
+----
+#pmdahotproc
```

Comment:
Asking here, but applies in other locations. Is this worth shipping in some format? (e.g. package, Ansible collection)

Reply:
While I do have these files in the demo setup with Ansible right now, the Ansible collection (operations) seem wrong, as people might want to use a different deployment method.

Shipping a package that touches `/var/lib/pcp/` also feels icky, but we could ship those in `/usr/share/foreman/extras/` and ask people to symlink/copy from there?

Reply:
https://github.com/theforeman/foreman/pull/9853